examples/rc2.py

```diff
@@ -248,6 +248,12 @@ def __init__(self, formula, solver='g3', adapt=False, exhaust=False,
         if not formula.hard and len(self.sels) > 100000 and min(wght) == max(wght):
             self.minz = False
 
+    def _oracle_solve(self, **kwargs):
+        """
+            Calls `oracle.solve(assumptions)` with possible interrupts
+        """
+        return self.oracle.solve_limited(expect_interrupt=self.expect_interrupt)
```

Comment:
Make assumptions clearly passed through the arguments of `_oracle_solve()` and then add them when making the actual call of `solve_limited()` in line 255. Not sure how this should work without assumptions to be honest.